std/machines/hash/keccakf32_memory.asm

```diff
@@ -592,22 +592,19 @@ machine Keccakf32Memory(mem: Memory) with
     //     }
     // }
 
-    let query_c: int, int, int -> int = query |x, limb, bit_in_limb|
-        utils::fold(
-            5, 
-            |y| (int(eval(a[y * 10 + x * 2 + limb])) >> bit_in_limb) & 0x1, 
-            0, 
-            |acc, e| acc ^ e
-        );
-
     query |row| {
```

Comment:
I think this could be something like
```
query |row| compute_from_multi(c, row, a, |a_values| {
  array::new(array::len(c), |i| {
     let x = i / 64;
     let z = i % 64;
     let limb = z / 32;
     let bit_in_limb = z % 32;
//      compute c here
  })
});
```